Core/GDCore/Events/CodeGeneration/EventsCodeGenerationContext.h

```diff
@@ -260,6 +292,13 @@ class GD_CORE_API EventsCodeGenerationContext {
                                       ///< but not filled with scene's
                                       ///< objects and not filled with any
                                       ///< previously existing objects list.
+  std::set<gd::String>
+      allListsAcrossChildrenToBeDeclared;  ///< This is only to be used by the

```

Comment:
Naming is not clear to me: why not `allObjectsAcrossChildrenToBeDeclared` ?

Reply:
What do you think about `allObjectsListToBeDeclaredAcrossChildren`?

Reply:
`allObjectsListToBeDeclaredAcrossChildren`: this seems the most exact naming 👍